### ISSUE TITLE
Fail to compile with SBCL 1.3.2 x86-64 Linux

### ISSUE THREAD

**Issue by cmpitg** [cosmos72/stmx#12]
Edited: Sorry, it only happened to SBCL 1.3.0.

Please find below the stack trace I got:

```
; Loading "stmx"
; 
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Lock on package SB-VM violated when interning OFFSET-NEXT while in package
;     SB-TRANSACTION.
;   See also:
;     The SBCL Manual, Node "Package Locks"
;   
;     (in form starting at line: 18, column: 0, position: 601)

debugger invoked on a UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread
#<THREAD "main thread" RUNNING {1002A4C6F3}>:
  COMPILE-FILE-ERROR while
  compiling #<CL-SOURCE-FILE "stmx" "sb-transaction" "x86-64-insts">

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [RETRY                        ] Retry
                                     compiling #<CL-SOURCE-FILE "stmx" "sb-transaction" "x86-64-insts">.
  1: [ACCEPT                       ] Continue, treating
                                     compiling #<CL-SOURCE-FILE "stmx" "sb-transaction" "x86-64-insts">
                                     as having been successful.
  2:                                 Retry ASDF operation.
  3: [CLEAR-CONFIGURATION-AND-RETRY] Retry ASDF operation after resetting the
                                     configuration.
  4: [ABORT                        ] Give up on "stmx"
  5:                                 Exit debugger, returning to top level.

(UIOP/LISP-BUILD:CHECK-LISP-COMPILE-RESULTS NIL T T "~/asdf-action::format-action/" ((#<ASDF/LISP-ACTION:COMPILE-OP > . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "stmx" "sb-transaction" "x86-64-insts">)))
0] 
```

Any ideas how to fix it?

Ha-Duong

**Comment by cosmos72** [comment]
I guess you meant "it only happened to SBCL 1.3.2".
It's the same issue as #11, fixed in commit 183768d6aa995f5a529affeb899aed70a6463fc1

**Comment by cmpitg** [comment]
My bad. Yes I meant it happened to SBCL 1.3.2. Thanks for the fix. It works now.

Ha-Duong
